client.go

```diff
@@ -31,7 +28,7 @@ type Client struct {
 
 	client *client.Client
 
-	w       *ethwallet.Wallet
+	w       *Wallet
```

Comment:
After `NewWallet`, you only need the `ethwallet.Wallet`, so I think you can keep `w` as `ethwallet.Wallet` as it was before.